packages/appChrome/components/HeaderBar.tsx

```diff
@@ -1,15 +1,68 @@
 import * as React from "react";
 import { cx } from "emotion";
-import { darkMode, flex, padding } from "../../shared/styles/styleUtils";
+import {
+  background,
+  darkMode,
+  flex,
+  padding
+} from "../../shared/styles/styleUtils";
 import { headerBar, appChromeInsetContent } from "../style";
 
 export interface HeaderProps {
+  backgroundColor?: string;
   children: React.ReactElement<HTMLElement> | string;
+  paddingBottom?:
+    | "xxs"
+    | "xs"
+    | "s"
+    | "m"
+    | "l"
+    | "xl"
+    | "xxl"
+    | "none"
+    | undefined;
+  paddingLeft?:
+    | "xxs"
+    | "xs"
+    | "s"
+    | "m"
+    | "l"
+    | "xl"
+    | "xxl"
+    | "none"
+    | undefined;
+  paddingRight?:
+    | "xxs"
+    | "xs"
+    | "s"
+    | "m"
+    | "l"
+    | "xl"
+    | "xxl"
+    | "none"
+    | undefined;
+  paddingTop?:
```

Comment:
Header bar height shouldn't be customizable according to the jira.

Reply:
I'm not suggesting letting the height be customizable, but if we have don't remove `height: 32px`, the top and bottom padding won't be accurate because it starts to take up the height. Here's two screenshots to demo what I'm talking about:
![paddingandheight](https://user-images.githubusercontent.com/2313998/49089945-af654d00-f22a-11e8-9985-9a42824611a5.gif)

<img width="180" alt="screen shot 2018-11-27 at 9 53 33 am" src="https://user-images.githubusercontent.com/2313998/49089930-a5dbe500-f22a-11e8-822c-add4c828d83c.png">